etc/dbus-serialbattery/reinstall-local.sh

```diff
@@ -339,14 +409,33 @@ if [ "$can_lenght" -gt 0 ]; then
     echo
 
     # install required packages
-    # TO DO: Check first if packages are already installed
-    echo "Installing required packages to use CAN connection..."
+    echo "Checking required packages to use CAN connection..."
+    echo
+
+    if ! opkg list-installed | grep -q "python3-misc" || ! opkg list-installed | grep -q "python3-pip"; then
+        echo "Update packages..."
+        opkg update
+        echo
+    fi
 
-    opkg update
-    opkg install python3-misc python3-pip
+    if ! opkg list-installed | grep -q "python3-misc"; then
+        echo "Install python3-misc..."
+        opkg install python3-misc
+        echo
+    fi
 
-    echo
-    pip3 install python-can
+    if ! opkg list-installed | grep -q "python3-pip"; then
+        echo "Install python3-pip..."
+        opkg install python3-pip
+        echo
+    fi
+
+    # fastest way to check if can is installed
+    if [ ! -f "/usr/lib/python3.8/site-packages/can/__init__.py" ]; then
+        echo "Install can..."
+        pip3 install can
```

Comment:
"can" package is "A color animator"  https://libraries.io/pypi/can

you probably wanted python-can (https://python-can.readthedocs.io/en/stable/)

Reply:
Thanks, I don‘t know how that worked then for others :-P